mapbox_gl_platform_interface/lib/src/mapbox_gl_platform_interface.dart

```diff
@@ -168,4 +171,9 @@ abstract class MapboxGlPlatform {
   Future<LatLngBounds> getVisibleRegion() async {
     throw UnimplementedError('getVisibleRegion() has not been implemented.');
   }
+
+  Future<LatLngBounds> addImage(String name, Uint8List bytes,
```

Comment:
```suggestion
  Future<null> addImage(String name, Uint8List bytes,
```
There's actually no return value for addImage(), this should solve the linting issue.